### PR TITLE
Removed references to icla/ccla configurations

### PR DESCRIPTION
**Pull request by msys-sgarg**
Signed-off-by: smriti <sgarg@msystechnologies.com>

### Description

Since support for ICLA/CCLA is withdrawn from supermarket.I have removed all the references to icla/ccla 

### Issues Resolved

https://github.com/chef/supermarket/issues/2152

### Check List

- [ ] New functionality includes tests
- [x] All buildkite tests pass
- [x] Full omnibus build and tests in buildkite pass
- [x] All commits have been signed-off for the Developer Certificate of Origin. See <https://github.com/chef/chef/blob/main/CONTRIBUTING.md#developer-certification-of-origin-dco>
- [x] PR title is a worthy inclusion in the CHANGELOG

### CODE REVIEW

File: omnibus/cookbooks/omnibus-supermarket/attributes/default.rb [hunk 1]
```diff
@@ -393,19 +393,8 @@
...
 default['supermarket']['chef_oauth2_verify_ssl'] = true
-
-# ### CLA Settings
-#
-# These are used for the Contributor License Agreement features. You only need
-# them if the cla and/or join_ccla features are enabled (see "Features" below.)
-default['supermarket']['ccla_version'] = nil
-default['supermarket']['cla_signature_notification_email'] = nil
-default['supermarket']['cla_report_email'] = nil
 default['supermarket']['curry_cla_location'] = nil
```
[the remark this answers is not in the document]
Comment (msys-sgarg) [reply omnibus/cookbooks/omnibus-supermarket/attributes/default.rb:396]: ok, will take care of that

**Comment by saghoshprogress** [comment]
@msys-sgarg  Buldkite build is broken. can you please check and fix the same?

**Comment by tas50** [comment]
Here's a failure on a duplicate branch I created. If you create branches directly on the repo it's easier to test them with adhocs:

https://buildkite.com/chef/chef-supermarket-main-omnibus-adhoc/builds/177#33542188-e0f3-48e8-bda1-d96acf3a611a

**Comment by msys-sgarg** [comment]
@tas50  ok thanks, will take care of this

**Pull request merged**